mungegithub/mungers/milestone-maintainer.go

```diff
@@ -451,17 +451,17 @@ func checkLabels(labels []githubapi.Label) (kindLabel, priorityLabel string, sig
 
 	kindLabel, err = uniqueLabelName(labels, kindMap)
 	if err != nil || len(kindLabel) == 0 {
-		labelErrors = append(labelErrors, "kind: Must specify at most one of ['kind/bug', 'kind/feature', 'kind/cleanup'].")
+		labelErrors = append(labelErrors, "_**kind**_: Must specify at most one of [`kind/bug`, `kind/feature`, `kind/cleanup`].")
```

Comment:
Shouldn't this be "Must specify **exactly** one of ..." instead of "Must specify **at most** one of ..."? This applies to `kind/` labels and `priority/` labels